src/main/scala/io/qbeast/spark/QbeastTable.scala

```diff
@@ -46,7 +45,7 @@ class QbeastTable private (
    * @param revisionID the identifier of the revision to optimize.
    *                          If doesn't exist or none is specified, would be the last available
    */
-  def optimize(revisionID: Option[RevisionID] = None): Unit = {
+  def optimize(revisionID: RevisionID = -1L): Unit = {
```

Comment:
Why the -1L, it is a bit ugly to use a specific number with a specific value. It was better before with the Option internally. 
I understand that having to do optimize(Some(23))  is ugly, but that can be solved by having two public methods optimize() and optimize(123) that map to the same private method. In the end, is what the scala does under the hood with default arguments.

Reply:
Yes, the reason was to not put Some(x), but two different public methods is a good catch